libraries/pushproviders/unifiedpush/build.gradle.kts

```diff
@@ -41,7 +41,13 @@ dependencies {
     implementation(libs.serialization.json)
 
     // UnifiedPush library
-    api(libs.unifiedpush)
+    implementation(libs.unifiedpush) {
+        // Exclude package com.google.crypto.tink
+        // Duplicate classes between
+        // tink-1.16.0.jar -> tink-1.16.0 (com.google.crypto.tink:tink:1.16.0)
+        // tink-android-1.8.0.jar tink-android-1.8.0 (com.google.crypto.tink:tink-android:1.8.0)
+        exclude(group = "com.google.crypto.tink", module = "tink")
```

Comment:
Hello @p1gp1g , do you know why I need to add this line?

It seems to be related to https://github.com/UnifiedPush/android-connector/blob/cf347ae931ea8b6e6c773cd566abea940d1297f1/connector/build.gradle#L58 and the related PR https://github.com/tink-crypto/tink-java-apps/pull/5, but maybe I miss something?

Reply:
Thanks for the fast and complete response. Maybe the unified push library should use `tink-android` and not `tink` and that would fix the conflicts? (even if I agree that we have another dependency (not checked which one yet) which has a dependency on an outdated version `1.8.0`).

Reply:
You would have the same issue if 2 dependencies rely on tink-android, isn't it ?

I have opened [an issue](https://codeberg.org/UnifiedPush/android-connector/issues/5) to migrate to tink-android, there are some pro/con to do the migration there

Reply:
> (even if I agree that we have another dependency (not checked which one yet) which has a dependency on an outdated version 1.8.0)

I think it's this one:

```
androidx.security:security-crypto:1.1.0-alpha06
+--- androidx.annotation:annotation:1.1.0 -> 1.9.1 (*)
+--- androidx.annotation:annotation:1.2.0 -> 1.9.1 (*)
+--- androidx.collection:collection:1.1.0 -> 1.4.4 (*)
\--- com.google.crypto.tink:tink-android:1.8.0
```

Reply:
Related: https://stackoverflow.com/questions/78362124 ...